application/F3DStarter.cxx

```diff
@@ -956,6 +960,31 @@ int F3DStarter::Start(int argc, char** argv)
       }
     }
 
+    // Process Command Script file
+    const std::string& commandScriptFile = 
+      this->Internals->AppOptions.CommandScriptFile;
+    if (!commandScriptFile.empty())
+    {
+      std::ifstream scriptFile(commandScriptFile);
+      if (scriptFile.is_open())
+      {
+        std::string command;
+        while (std::getline(scriptFile, command))
+        {
+          if (!command.empty() && command[0] != '#')
+          {
+            interactor.triggerCommand(command);
+          }
+        }
```

Comment:
else error out ?

Reply:
Do I error out if any of the commands in the script file are not valid?